advanced_filters/static/advanced-filters/advanced-filters.js

```diff
@@ -70,7 +70,11 @@ var OperatorHandlers = function($) {
 						  MODEL_LABEL) + '/' + field;
 		var input = $(elm).parents('tr').find('input.query-value');
 		input.select2("destroy");
+		var value = input.val();
 		$.get(choices_url, function(data) {
+			if (value) {
+				data.results.push({'id': value, 'text': value})
```

Comment:
The value actually might not exist in the results. One example is choices being disabled for a field.